src/Unity.Mathematics.CodeGen/VectorGenerator.cs

```diff
@@ -3295,55 +3295,43 @@ void GenerateRandomPerformanceTests(StringBuilder str)
             BeginPerformanceTestCodeGen(str, "TestRandom");
 
             GeneratePerformanceTest(str, "Random_NextUint", new PerformanceTestArrayArgument[] {
-                new PerformanceTestArrayArgument { m_ElementType = "Random", m_MemberName = "rng", m_ElementInitializer = "new Unity.Mathematics.Random(1)" },
                 new PerformanceTestArrayArgument { m_ElementType = "uint", m_MemberName = "u", m_ElementInitializer = "0" },
-            }, "args.u[i] = args.rng[i].NextUInt();", 10000);
+            }, "args.u[i] = args.rng.NextUInt();", 10000);
```

Comment:
These changes are for convenience. Prior to this PR, I had to create an array of `Random` to get random numbers, but I made it so the perf test code generator will just implicitly include an instance of `Random` to avoid having to create these arrays.